src/extension/tests/helpers/RuntimeComposer.py

```diff
@@ -9,10 +9,9 @@
 class RuntimeComposer(object):
     def __init__(self):
         self.logger = Logger()
-        telemetry_writer = TelemetryWriter()
-        self.logger = Logger(telemetry_writer=telemetry_writer)
-        self.utility = Utility(self.logger)
-        self.json_file_handler = JsonFileHandler(self.logger)
+        self.telemetry_writer = TelemetryWriter(self.logger)
+        self.utility = Utility(self.logger, self.telemetry_writer)
```

Comment:
Not for this PR: Utility is being woven in too deep, and moving in a direction away from the final goal sitting in the backlog.